Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -5319,6 +5594,285 @@ def list_siteshield_maps_command(client: Client) -> Tuple[str, Dict[str, Any], U
     return human_readable, context_entry, raw_response
 
 
+# Created by D.S. 2023-05-03
+@logger
+def get_cps_enrollment_deployment_command(client: Client,
+                                          enrollment_id: int,
+                                          environment: str = 'production',) -> Tuple[str, Dict[str, Any], Union[List, Dict]]:
+    """
+        Returns the certification/Enarollment deployment status for specific a environtment: production or staging.
+
+    Args:
+        client:
+        enrollment_id: Unique Identifier of the Enrollment on which to perform the desired operation.
+            And it can be retrived via list_enrollments_command
+        environment: Environment where the certificate is deployed: production or staging
+
+    Returns:
+        human readable (markdown format), entry context and raw response
+    """
+
+    raw_response: Dict = client.get_cps_enrollment_deployment(enrollment_id=enrollment_id, environment=environment)
+
+    title = f'{INTEGRATION_NAME} - get cps enrollment deployment command'
+    entry_context = raw_response
+    human_readable_ec = raw_response
+    context_entry: Dict = {
+        f"{INTEGRATION_CONTEXT_NAME}.Cps.Enrollments.Deployment": entry_context
+    }
+
+    human_readable = tableToMarkdown(
+        name=title,
+        t=human_readable_ec,
+        removeNull=True,
+    )
```

Comment:
```suggestion
    )
    demisto.debug(f'{human_readable=} , {context_entry=} , {raw_response}')
```
Please add this demisto.debug line to all of the new commands, so it will be easier to debug them with the logs.